src/WebJobs.Script.Grpc/Server/IHttpProxyService.cs

```diff
@@ -11,6 +11,11 @@ public interface IHttpProxyService
     {
         Task EnsureSuccessfulForwardingAsync(ScriptInvocationContext context);
 
-        Task ForwardAsync(ScriptInvocationContext context, Uri httpUri);
+        /// <summary>
+        /// Initiates a request forward and updates the current context to track forwarding operation.
+        /// </summary>
+        /// <param name="context">The <see cref="ScriptInvocationContext"/> for the HTTP invocation.</param>
+        /// <param name="httpUri">The target URI used for forwarding.</param>
+        void StartForwarding(ScriptInvocationContext context, Uri httpUri);
```

Comment:
Is it a breaking change to change a public interface?

Reply:
For publicly exposed assemblies, yes, but these are not publicly consumed/published APIs. Those changes are not uncommon in the host assemblies.

Reply:
Happy the question was asked, though :D